docs_src/general/ServiceConfiguration.md

```diff
@@ -0,0 +1,38 @@
+# Service Configuration
```

Comment:
This is the common config section that is linked to from each the services config section. All those link need to be updated.

Reply:
Not sure I understand Lenny.  I did update them all.
I can even eliminate this page, but I thought it would be convenient to have a single place to link in to all the configuration sections of each micro service.  So areas may change over time.  This will happen in all sorts of places.

Reply:
All the config sections for each service has a link back to this common config document. For app serivecs:
```

Reply:
ah - now I see what you are referencing - all the hard links to the 1.2 docs.  All better now.  Good catch.